frameworks/java/prewarm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "loading" > /workspace/prewarm.status
+
+# prewarm by starting the gradle daemon. Running an initial test build will also speed things up a bit
+cd /runner/frameworks/java && gradle --daemon --offline test
```

Comment:
How bad was the performance of simple `javac` and `java`?
I can see that `jvm-runner` was slow because of Clojure+Leiningen startup, but I've been able to compile and run Java in decent speed for my side project.

I can just `docker run` and this finishes in few seconds (multiple files and Codewars output).
```sh
# Takes JSON from STDIN. `yaml r -j` converts YAML to JSON.
docker run --rm -i --memory=1GB java-runner:openjdk8 <<< $(yaml r -j test_junit.yml)
```